docs/en/guides/compile/compile.md

```diff
@@ -1,7 +1,7 @@
 # Compiling
 
 ## Platform
-Linux and MacOS are supported in SkyWalking Satellite. Windows is not due to following components. Please read [the doc](../../FAQ/running_on_windows.md).
+Linux and MacOS are supported in SkyWalking Satellite. Windows is not due to following components. 
```

Comment:
Is Windows unsupported anyway, or do we have a fallback queue implementation on Windows? For example, users can try it on Windows with `memory-queue`?

Reply:
already push a memory queue

Reply:
> already push a memory queue

I know that, I mean we cannot simply say "Windows is not due to following components.", `mmap-queue` is only a component in Satellite, users can still try / use Satellite if they configure another component (like `memory-queue`), the unavailability is component level and should be documented in that specific component, i.e. document "This component is not available on Windows" in [the `mmap-queue` plugin doc](https://github.com/apache/skywalking-satellite/blob/main/docs/plugins/queue/mmap/README.md), instead of here, "compile" doc.

Reply:
This is the question in my mind, if memory queue is available, then satellite should be available for WIndows platform. We should be clear about that. Could you deliver the window version binary? The current idea is saying code change, which is not a good one.

Reply:
fixed

Reply:
zhenxu help me to unify the compile at all platforms. Thanks a lot.